agent/agent.go

```diff
@@ -152,6 +152,7 @@ func (r *Agent) Run(recipe recipe.Recipe) (run Run) {
 	// to gather total number of records extracted
 	stream.setMiddleware(func(src models.Record) (models.Record, error) {
 		recordCount++
+		r.logExtractedRecord(recipe, src)
```

Comment:
```suggestion
		r.logger.Info("Successfully extracted record", "record", src.Data().GetResource().Urn, "recipe", recipe.Name)
```

can we do like this directly?